examples/next-openai/app/use-chat-attachments/page.tsx

```diff
@@ -6,10 +6,9 @@ import { useChat } from 'ai/react';
 import { useRef, useState } from 'react';
 
 export default function Page() {
-  const { messages, input, handleSubmit, handleInputChange, isLoading } =
-    useChat({
-      api: '/api/chat',
-    });
+  const { messages, input, setInput, append, isLoading } = useChat({
```

Comment:
should we update this example or leave it as is (and add a 2nd example if needed)?

Reply:
makes sense to add a 2nd example in case this was linked from somewhere!